smartapps/konnected-io/konnected-security-connect.src/konnected-security-connect.groovy

```diff
@@ -127,11 +126,7 @@ def pageDiscovery() {
   //create accessToken
   if(!state.accessToken) { createAccessToken() }
 
-  //This is a workaround to prevent page to refresh too fast.
-  if(!state.pageConfigurationRefresh) { state.pageConfigurationRefresh = 2 }
-
-  dynamicPage(name: "pageDiscovery", nextPage: "pageConfiguration", refreshInterval: state.pageConfigurationRefresh) {
-    state.pageConfigurationRefresh =  state.pageConfigurationRefresh + 3
```

Comment:
I removed this incrementing refresh interval because every time you load the Discovery page, it would add 3 seconds. If you do this several times, the refresh interval is too long and it looks like it's not working. Any reason that a hardcoded 4-sec value won't work here?